packages/common/src/services/remote-config/remote-config.ts

```diff
@@ -219,8 +221,10 @@ export const remoteConfig = <
 
     try {
       const enabled = state.didInitialize
-        ? client.isFeatureEnabled(flag, id.toString(), { userId: id }) ??
-          defaultVal
+        ? client.isFeatureEnabled(flag, id.toString(), {
+            userId: id,
+            appVersion
```

Comment:
woah so this is just an allowed value with optimizely in these later versions? clutch

Reply:
We can actually pass any value, but yeah the new versions add a `version` type that can be matched on via semver